src/site/stages/build/drupal/page.js

```diff
@@ -6,7 +6,12 @@ const set = require('lodash/fp/set');
 // Creates the file object to add to the file list using the page and layout
 function createFileObj(page, layout) {
   // Exclude some types from sitemap.
-  const privateTypes = ['outreach_asset', 'support_service'];
+  // @todo remove basic_landing_page when /resources/ is ready to be indexed
+  const privateTypes = [
+    'outreach_asset',
+    'support_service',
+    'basic_landing_page',
```

Comment:
This change is actually a follow up to https://github.com/department-of-veterans-affairs/vets-website/pull/14832. It adds a new type of layout to the ignore list.